content/integrations/integrating-npm-with-external-services/docker-and-private-modules.mdx

```diff
@@ -79,7 +79,11 @@ docker build --build-arg NPM_TOKEN=${NPM_TOKEN} .
 
 This will build the Docker image with the current `NPM_TOKEN` environment variable, so you can run `npm install` inside your container as the current logged-in user.
 
+To set your `NPM_TOKEN` environment varible run `export NPM_TOKEN="00000000-0000-0000-0000-000000000000"` and pass your npm access token instead of `0`s.
+
 <Note>
+  
+**Note:** After setting your `NPM_TOKEN` environment variable remember to clean your terminal history. You can do it with `history -c` command for `bash` and `history -p` for `zsh`.
```

Comment:
This is a dangerous command to be blindly recommending; terminal history is important.

Reply:
Hey @ljharb could you please give some examples of when it's important? I'm not opposing. I'm genuinely interested.

Reply:
It's me again @ljharb I've updated the PR as I think my current PR does not solve the issue but I've outlined a solution which I think might be what we need.